business/apps.go

```diff
@@ -124,43 +117,26 @@ func (in *AppService) GetAppList(namespace string, linkIstioResources bool) (mod
 		for _, srv := range valueApp.Services {
 			joinMap(applabels, srv.Labels)
 			if linkIstioResources {
-				svcVirtualServices := kubernetes.FilterVirtualServices(*linkedResources[kubernetes.VirtualServices], srv.Namespace, srv.Name)
-				svcDestinationRules := kubernetes.FilterDestinationRules(*linkedResources[kubernetes.DestinationRules], srv.Namespace, srv.Name)
-				allFiltered := append(svcVirtualServices, svcDestinationRules...)
```

Comment:
Here I would keep this appending, as more object types could come to be shown.

Reply:
Note that the work is still in WIP and there are some refactoring that won't change the logic but some implementation detail.
Some of the decisions is to consolidate the Istio fetch using the business package.

The allFiltered is an array/slice of IstioObject and that struct/interface doesn't exist anymore.
So, same logic is done below, appending per type.